site/src/Map.jsx

```diff
@@ -340,6 +352,7 @@ export default function Map({
 
           <NavigationControl position="top-right"></NavigationControl>
           <GeolocateControl />
+          <ScaleControl position="bottom-left" />
```

Comment:
Default unit is metric. Can we add a function with a button to allow users to toggle between metric and imperial?